azure-storage-cpp-lite/src/blob/blob_client_wrapper.cpp

```diff
@@ -149,7 +149,7 @@ namespace microsoft_azure {
             }
             catch(const std::exception &ex)
             {
-                std::cerr << ex.what() << std::endl;
+                syslog(LOG_ERR, "Unknown failure in blob_client_wrapper_init.  ex.what() = %s.", ex.what());
```

Comment:
It is not unknown. The exact error was catched. Just say `"Failed to create blob client: %s", ex.what()`

Reply:
Thanks!  Latest commit changes it to: 
syslog(LOG_ERR, "Failed to create blob client.  ex.what() = %s.", ex.what());